### PR TITLE
XMPP Summit 27

### PR DESCRIPTION
**Pull request by nemmig**
I know it isn't much but its honest work ...
This file has been created using  /xmpp.org/content/blog/2024-01-05-xmpp-summit-26.md as a template.
Hope you guys find it useful.

### CODE REVIEW

**Comment by Echolon** [comment]
Great work @nemmig !

**Comment by nemmig** [comment]
> Great work @nemmig !

Thank you very much!
You are welcome! :hugs:

File: content/blog/2025-01-05-xmpp-summit-27.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
+---
+title: XMPP Summit 27
+date: 2025-01-05
```
Comment (cal0pteryx) [inline content/blog/2025-01-05-xmpp-summit-27.md:3]: This post should be released as soon as possible, and the text should be adapted accordingly (i.e. the summit being next year, 2025).

Comment (Echolon) [reply content/blog/2025-01-05-xmpp-summit-27.md]: 7bd117d

**Comment by Echolon** [comment]
@cal0pteryx thank you very much for review! <3

**Pull request merged**